### PR TITLE
Turkish language support has been added

### PR DESCRIPTION
**Pull request by YTzcn**
tr.json has been added to LocalizationData.json, Turkish option has been added to LanguageFlags folder, tr.png has been added.

### CODE REVIEW

**Review by CPKreuz — CHANGES_REQUESTED** [review]
Hi! Thanks for adding Turkish!

* Currently, the tr.png is not marked as a Resource. If you can open the project using VS you can set the build action to Resource in the tr.png file's properties or you can also add the following two lines in the src/PixiEditor/PixiEditor.csproj file after the line 424

```xml
<None Remove="Images\LanguageFlags\tr.png" />
<Resource Include="Images\LanguageFlags\tr.png" />
```

* Please also replace the tr.png with [this](https://github.com/PixiEditor/PixiEditor/assets/45312141/d765dea1-4b22-4259-b78d-3ae7583d6868) one from icons8



---

Also, it would be cool if you could join our [Discord](https://discord.gg/tzkQFDkqQS) server so you can join our Translator team to get access to POEditor where you can edit the language files in a web page

**Review by YTzcn — COMMENTED** [review]
Okay, I understand. I hope I was able to do it :)

File: src/PixiEditor/Data/Localization/LocalizationData.json [hunk 1]
```diff
@@ -85,6 +85,13 @@
       "localeFileName": "it.json",
       "iconFileName": "it.png",
       "lastUpdated": "2023-06-25 14:44:38"
+    },
+    {
+      "name": "Türkçe",
+      "code": "tr",
+      "localeFileName": "tr.json",
+      "iconFileName": "tr.png",
+      "lastUpdated": "2023-06-29 12:32:37"
```
Comment (CPKreuz) [inline src/PixiEditor/Data/Localization/LocalizationData.json:94]: That's the time from POEditor

```suggestion
      "lastUpdated": "2023-06-25 19:43:24"
```

File: src/PixiEditor/PixiEditor.csproj [hunk 2]
```diff
@@ -422,6 +422,8 @@
 		<Resource Include="Images\LanguageFlags\ru.png" />
 		<None Remove="Images\LanguageFlags\cs.png" />
 		<Resource Include="Images\LanguageFlags\cs.png" />
+    <None Remove="Images\LanguageFlags\tr.png" />
+    <Resource Include="Images\LanguageFlags\tr.png" />
```
Comment (CPKreuz) [inline src/PixiEditor/PixiEditor.csproj:426]: Let's keep the indentation 

```suggestion
		<None Remove="Images\LanguageFlags\tr.png" />
		<Resource Include="Images\LanguageFlags\tr.png" />
```

**Review by CPKreuz — CHANGES_REQUESTED** [review]
Looking good now!

Just two more small things so we can merge

File: src/PixiEditor/Data/Localization/LocalizationData.json (hunk 1 above)
Comment (YTzcn) [reply src/PixiEditor/Data/Localization/LocalizationData.json:94]: I understood that, but...

File: src/PixiEditor/PixiEditor.csproj (hunk 2 above)
Comment (YTzcn) [reply src/PixiEditor/PixiEditor.csproj:426]: I couldn't understand this part everything looks fine
[image](https://github.com/PixiEditor/PixiEditor/assets/25502590/bfeafd5b-5ab3-4204-9860-fe01447d5e5d)

Comment (YTzcn) [reply src/PixiEditor/PixiEditor.csproj]: Okay ı solved :)

**Comment by CPKreuz** [comment]
POEditor shows some [QA Errors](https://poeditor.com/projects/qa?id_language=169&id=400351&ref_language=43), would be great if you could check them out and fix them if needed

**Comment by YTzcn** [comment]
> POEditor shows some [QA Errors](https://poeditor.com/projects/qa?id_language=169&id=400351&ref_language=43), would be great if you could check them out and fix them if needed

Ok I'm working on it. But it may take some time for me to understand the problem because it's my first time using poeditor.

**Review by CPKreuz — APPROVED** [review]
Lookin good!

**Pull request merged**